app/src/main/java/net/squanchy/venue/VenueInfoService.java

```diff
@@ -0,0 +1,19 @@
+package net.squanchy.venue;
+
+import net.squanchy.venue.domain.view.Venue;
+import net.squanchy.service.repository.VenueRepository;
+
+import io.reactivex.Observable;
+
+class VenueInfoService {
+
+    private final VenueRepository venueRepository;
+
+    VenueInfoService(VenueRepository venueRepository) {
+        this.venueRepository = venueRepository;
+    }
+
+    public Observable<Venue> venue() {
+        return venueRepository.venue();
```

Comment:
❌ need to ensure the user is logged in (as anon user at least). See how other services do it.